pkg/controller/node/nodecontroller.go

```diff
@@ -573,6 +668,42 @@ func (nc *NodeController) monitorNodeStatus() error {
 	return nil
 }
 
+// evictIntolerablePods checks for given node if there's any pod that
```

Comment:
@gmarek 
> I'm probably missing something, but I don't see how we cancel eviction in a case when Node comes back.

Do you mean cancelling the pod evictions when Node comes back?

I guess there is a race here, evictIntolerablePods() may not be called since node is removed from `zonePodEvictor[node.Zone]`, probably should call `evictIntolerablePods()` right before `cancelPodEviction()` when a node comes back.